src/pages/AddProject.tsx

```diff
@@ -14,20 +15,26 @@ type FormState = {
   currentLeadPosition: string,
   githubRepo: string,
   lookingFor: string,
-  // techStack: string,
+  techStack: { value: string, label: string }[],
   errors: {
     experienceLevel: string,
     currentLeadPosition: string,
     githubRepo: string,
     lookingFor: string,
-    // techStack: string,
+    techStack: string,
   },
   serverError: {
     message: string
   },
   cancelModalOpen: boolean
 }
 
+const techStackOptions = [{value: 'django', label: 'Django'},
```

Comment:
Let's add a comment above that this information should be coming from `getTagsForProjectID` API once #118 is finished.

Reply:
Done!